src/Avengers.js

```diff
@@ -0,0 +1,69 @@
+import './Avengers.css';
+import axios from 'axios';
+import { useEffect, useState } from "react";
+import AvengersDisplayPhotos from './AvengersDisplayPhotos';
+import GoBack from './GoBack';
+
+const Avengers = () => {
+
+    //call useState hook to hold the array back from my api - destructure variable and function 
+    const [charData, setCharData] = useState(null); //expecting an array of objects, so I initialize with null because it doesn't exist until user selection
```

Comment:
Not sure about this, but if we're expecting an array of objects, should we have an empty array as our initialize state instead of null?